Aagam_Shah/postgreAssignment/src/controller/school.controller.ts

```diff
@@ -0,0 +1,359 @@
+const db = require('../config/database').pool;
+import { Context } from "vm";
+
+function page_validation(page: number, size: number): {status: number, message: string}{
+  if (typeof page !== "number" || typeof size !== "number") {
+      return {status: 400,  message: "Invalid parameters"};
+  }
+  if(page < 0 || size <= 0){
+      return {status: 400, message: "Invalid parameters"};
+  }
+  return {status: 200, message: "true"};
+}
+
+function isValidId(uuid: string){
+  if( uuid.match(/^[0-9A-F]{8}-[0-9A-F]{4}-[4][0-9A-F]{3}-[89AB][0-9A-F]{3}-[0-9A-F]{12}$/i) )
+      return true;
+  return false;        
+}
+
+exports.getById = async (ctx: Context) => {
```

Comment:
lets not keep this generic function, it very dangerous

Reply:
removed